std_msgs/String task
---
std_msgs/Bool success
std_msgs/String error_message
